x-pack/plugins/ml/public/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$mchColorCritical: #fe5050;
```

Comment:
Hey Dave, it would be great if we could use `ml` as a prefix instead of `mch` as a prefix if there's no technical reason to have a 3 letter prefix instead of 2 letters. `ml` is used across orgs and projects and it feels a bit strange to have a different prefix just for scss. Also the localization project started this month and is using `ml` too.

Reply:
There's no technical reason, but we've maintained the same three letter prefix schema across every other CSS conversion in Kibana. I'd like to keep everything on pattern if possible, though I don't care what those three letters are(kml, eml...etc).

In a later PR I'd like to see all of machine learning's selectors to adopt the prefix as well to avoid the cascade going on in there now

Reply:
> There's no technical reason...

That's great to hear. Please can we retain our namespace `ml` as it is used throughout our universe.

Reply:
I know better to get in arguments about naming! Here's a ticket for a later PR. Implement it however you feel you need to and change it however you see fit. 😄 

If you help doing that conversion after this one let me know.

https://github.com/elastic/kibana/issues/25691